.github/workflows/main.yml

```diff
@@ -79,6 +80,33 @@ jobs:
           path: ./*
           key: ${{ github.ref }}-${{ github.sha }}-build-release
 
+  build-docs:
+    needs: setup
```

Comment:
Since this job doesn't need any of the JavaScript stuff (as far as I know), I think we can remove the dependency on `setup` and check out the repo instead. This should allow the job to start earlier, and reduce the time it takes for the CI to complete.

```suggestion
```

Reply:
Done 🙏